www/_posts/2019-03-24-nodejs-6.x-8.x-deprecation-timeline.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title: "Node 6.x and 8.x Deprecation Timeline for Apache Cordova"
+categories: news
+tags: deprecation timeline node
+---
+
+Currently, all of the latest released platforms, tools, and core libraries require a minimum version of `node 6.x` to be installed. Additionally, we have added support for `node 10.x`.
+
+The Node.js Foundation has reported that the End-of-Life (EOL) for version 6.x will be April 30th, 2019. Looking ahead, the Node.JS Foundation has also scheduled the deprecation of 8.x on December 31, 2019 to be inaligned with the scheduled End-of-Life of OpenSSL-1.0.2.
```

Comment:
"inaligned" isn't a word. Did you mean "aligned" or "in alignment"?

Reply:
Yes, `aligned` was what I meant to write.